versions.yml

```diff
@@ -2,6 +2,16 @@
 logstash: 7.0.0-alpha1
 logstash-core: 7.0.0-alpha1
 logstash-core-plugin-api: 2.1.16
+
+# jruby must reference a *released* version of jruby which can be downloaded from the official download url
+# *and* for which jars artifacts are published for compile-time
 jruby:
   version: 9.1.12.0
   sha1: 9b6c15d42eb11db7215d94dd9842ee5184a41fea
+
+# jruby-runtime, if specified, will override the jruby version installed in vendor/jruby for logstash runtime only,
+# not for the compile-time jars
+jruby-runtime:
+  url: http://ci.jruby.org/snapshots/previous/jruby-bin-9.1.13.0-SNAPSHOT.tar.gz
```

Comment:
Did you mean to include this? It seems like including this file in master (or any other branch) will cause JRuby snapshots to be the only version of jruby used during the build?

Reply:
yes. the current released version is broken (JIT problem) and we want to start testing against the next version with will be released sometimes in the future which already includes the fix in the branch. this allows  the build to pass hopefully and once the version we want exists then we can remove the `jruby-runtime`. this is intended to be a temporary setting only when we specifically need/want to test agains a snapshot version.